WORKSPACE

```diff
@@ -353,7 +353,7 @@ php(
 php_gapic_repositories()
 
 # PHP micro-generator (beta)
-_gapic_generator_php_version = "0.1.1"
+_gapic_generator_php_version = "0.1.2"
```

Comment:
It is Ok for this PR, but please also update it in googleapis to keep the two workspace files in sync